spec/controllers/copy_course_from_production_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe CopyCourseFromProductionController, type: :request do
+    describe '#copy' do 
+        let(:user) { create(:user) }
+        let(:admin) { create(:admin) }
+        let(:existent_prod_course) { 'https://dashboard.wikiedu.org/courses/University_of_South_Carolina/Invertebrate_Zoology_(Spring_2022)' }
+
+        let(:subject) do
+            copy route, params: { url: existent_prod_course }
+        end
+
+        context 'when the user is an admin' do
+            before do
+                allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+            end
+
+            it 'copies the course' do
+                subject
+                expect(response.status).to eq(200)
+                expect(Course.exists?(:title => existent_prod_course)).to eq(true)
```

Comment:
This doesn't make sense, as `existent_prod_course` is a URL string, not a course title (which in this case, is "Invertebrate Zoology").

Reply:
This probably also needs to stub `CopyCourseFromProduction`, as we don't want to hit a production server more than once at most in the test suite. (We don't currently hit production dashboard servers at all in the test suite, and doing so makes me just a little bit uneasy, but I think it makes sense for a test of the service itself.)

Reply:
I did the change using slug property to check the existence [d75a150](https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/4914/commits/d75a1504d60105e3a1ad5da485c735e90c16cd37)

Reply:
I moved the test case to check service only from CopyCourseFromProduction [d49f462](https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/4914/commits/d49f46200acd3cf91310105c9ff01dd8c7c77940)